lib/ParseSIL/ParseSIL.cpp

```diff
@@ -1327,7 +1327,7 @@ bool SILParser::parseSILDeclRef(SILDeclRef &Result,
 
   // Construct SILDeclRef.
   Result = SILDeclRef(VD, Kind, expansion, /*isCurried=*/false, IsObjC);
-  if (uncurryLevel < Result.getUncurryLevel())
+  if (uncurryLevel < Result.getParameterListCount() - 1)
```

Comment:
Instead of using a constant - 1 here, can you turn this into a method on Result with a descriptive name/comment.